library/std/src/sys/unix/process/process_unix.rs

```diff
@@ -533,3 +536,7 @@ impl fmt::Display for ExitStatus {
         }
     }
 }
+
+#[cfg(test)]
+#[path = "process_unix/tests.rs"]
+mod tests;
```

Comment:
Is this path attribute really needed ?
In normal, tests file will be found under `<module name>/tests.rs`.

Reply:
Yes, due to this module being included via a path attribute as well.